bin/restore-database-backups.sh

```diff
@@ -0,0 +1,14 @@
+#
+# Run this script to loop over .sql files inside ./database-backups
+# and restore their databases to the running mysql-business-analytics container
+#
+
+function restoreDatabaseBackup() {
+   echo "Processing $1"
+   cat "$1" | docker exec -i mysql-business-analytics /usr/bin/mysql --user=root --password=root
+}
+
+for i in database-backups/*
```

Comment:
perhaps you should restrict the files you loop over to only `*.sql` files?
Also if there are several backups of the same mysql database in the `database-backups` directory what happens? (I'm just wondering if attempting to restore **every** backup file in the directory is going to have the desired result.

Reply:
Okay made that restriction. I tested with 3 of the same backup, and it just overwrites it, which I think would be the desirable? If you're uploading two backups with a database of the same name, I think that is a user mistake.

Reply:
So I guess I'm wondering about what backups w/ different databases we have that you are anticipating being restored.
I'm not suggesting you change this working code, just curious what future scenarios you were planning for. If you hadn't really given it that much thought, I'm just pointing out that it is worth thinking about as you're doing something like this.

Reply:
I guess I was anticipating there would be more than one client using our mattermost integration (edu), and that we could use different metabase dashboards for each client's database backup.

Reply:
I see, that makes sense, except you didn't think about the fact that the database name for every client is the same, so they will overwrite one another. But that is perhaps a future enhancement to for this repo to deal with.

Reply:
I did consider that, and knew the caveat was that the database names would have to be different.  Perhaps this should handle that gracefully and alternatively name the databases at run-time, or at the very least not overwrite them. I guess, like you said, this might be a future enhancement. 
Also, I think during my initial approach, my thinking was that this made things dynamic so I didn't have to hard-code a filename a .sql file to import.

Reply:
>  this made things dynamic so I didn't have to hard-code a filename
That is true and makes a lot of sense.